<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkOpenCVVideoControls</class>
 <widget class="QWidget" name="QmitkOpenCVVideoControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QRadioButton" name="UseGrabbingDeviceButton">
     <property name="text">
      <string>Use Grabbing Device</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="GrabbingDevicePanel" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Grabbing Device Number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="GrabbingDeviceNumber"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="UseVideoFileButton">
     <property name="text">
      <string>Use Video File</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="VideoFilePanel" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QmitkFileChooser" name="FileChooser" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="UpdateRatePanel" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Update rate [Hz]:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="UpdateRate">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="VideoFileControls" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QCheckBox" name="RepeatVideoButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Repeat Video</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="VideoProgressSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="PlayButton">
        <property name="text">
         <string>Play</string>
        </property>
        <property name="icon">
         <iconset resource="M:/workspace/mitk/src/mitk/Modules/OpenCVVideoSupport/UI/OpenCVVideoSupportUI.qrc">
          <normaloff>:/OpenCVVideoSupportUI/media-playback-start.png</normaloff>:/OpenCVVideoSupportUI/media-playback-start.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="StopButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="icon">
         <iconset resource="M:/workspace/mitk/src/mitk/Modules/OpenCVVideoSupport/UI/OpenCVVideoSupportUI.qrc">
          <normaloff>:/OpenCVVideoSupportUI/media-playback-stop.png</normaloff>:/OpenCVVideoSupportUI/media-playback-stop.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkFileChooser</class>
   <extends>QWidget</extends>
   <header location="global">QmitkFileChooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="M:/workspace/mitk/src/mitk/Modules/OpenCVVideoSupport/UI/OpenCVVideoSupportUI.qrc"/>
 </resources>
 <connections/>
</ui>
